### PR TITLE
Adds hreflang attribute to language version links

### PR DESCRIPTION
**Pull request by tommorris**
This helps search engines and other user agents to find different language versions of the page.

### CODE REVIEW

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 					</p>
 					<p>
 						{{ $lang := .Lang }}{{ $base := .Site.BaseURL }}{{ range .Site.Languages }}
-						<a class="nav-link" href="{{ $base }}{{ .Lang }}">{{ .LanguageName }}</a>
+						<a class="nav-link" href="{{ $base }}{{ .Lang }}" hreflang="{{ .Lang }}">{{ .LanguageName }}</a>
```
Comment (zeripath) [inline layouts/partials/footer.html:12]: AFIAU hreflang should only be used on `<link>`  I think what you need is lang, as it is on default template.

```suggestion
						<a class="nav-link" href="{{ $base }}{{ .Lang }}" lang="{{ .Lang }}">{{ .LanguageName }}</a>
```

Comment (tommorris) [reply layouts/partials/footer.html:12]: `<a href="https://en.wikipedia.org/wiki/Ciao" lang="it">Ciao</a>` means the text inside the element is in Italian. `lang` specifies the language of the text content; `hreflang` specifies the language of the target document.

`hreflang` [can be used just fine on `a` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-hreflang).

Comment (techknowlogick) [reply layouts/partials/footer.html:12]: Does google understand the full language written out (I mean probably as they are google after all, but maybe some smaller search engines may not)  my understanding is that we should use the form en-US etc for hreflang.

Comment (tommorris) [reply layouts/partials/footer.html:12]: For either a `lang` or `hreflang` attribute, it should just be the ISO language code.

`.Lang` should give that code, not `.LanguageName`.

**Review by techknowlogick — APPROVED** [review]
Thanks for PR and taking the time to answer feedback

File: layouts/partials/footer.html (hunk 1 above)
Comment (zeripath) [reply layouts/partials/footer.html:12]: Ok I guess it should have both then.

The language is the language name written in itself.

**Review by lafriks — APPROVED** [review]

**Pull request merged**